lib/rules/jest/no-if.js

```diff
@@ -25,63 +25,60 @@ module.exports = {
         'This is usually an indication that you',
         'are attempting to test too much at once',
         'or not testing what you intend to.',
-        'Consider breaking the conditional statement out',
-        'into a separate test to resolve this error.',
+        'Consider writing a separate test for',
+        'each fork in the conditional statement.',
```

Comment:
The other case I often see this: people have a possibly null value and bail out to not make TypeScript mad. We should call that one out, and push them to use `!` instead